test/cli.mjs

```diff
@@ -85,7 +85,8 @@ describe("postject CLI", () => {
         ],
         { encoding: "utf-8" }
       );
-      expect(stderr).to.be.empty;
+      // TODO(dsanders11) - Enable this once we squelch LIEF warnings
+      // expect(stderr).to.be.empty;
```

Comment:
Could you precise that it's on Linux on the comment, when I saw that it worked on my machine, I removed it.

Sorry again for this.

Reply:
@tony-go It might potentially happen on all platforms because LIEF has a lot more logs as you found out in https://github.com/nodejs/postject/pull/73, so might be better to keep this more generic?
Also, no worries! 🙌

Reply:
For example https://github.com/nodejs/postject/issues/75 on Windows

Reply:
> so might be better to keep this more generic?

Not sure, anyway we should be more careful for the next reviews ^^